cdap-docs/reference-manual/source/http-restful-api/introduction.rst

```diff
@@ -146,9 +146,12 @@ Status Codes
      - ``Not Implemented``
      - A request contained a query that is not supported by this API
 
-**Note:** These returned status codes are not necessarily included in the descriptions of the APIs,
-but a request may return any of these.
+**Notes** 
 
+- These returned status codes are not necessarily included in the descriptions of the APIs,
+  but a request may return any of these.
+- Requests can fail due to a lack of authorization or privilege, as described in the
```

Comment:
it may be better for this to go in the description of the 403 FORBIDDEN row in the table above?